config.toml

```diff
@@ -420,14 +432,14 @@ title = "Kubernetes"
 description = "Giải pháp điều phối container trong môi trường production"
 languageName = "Tiếng Việt"
 contentDir = "content/vi"
-weight = 12
+weight = 13
 languagedirection = "ltr"
 
 [languages.ru]
 title = "Kubernetes"
 description = "Первоклассная оркестрация контейнеров"
 languageName = "Русский"
-weight = 12
+weight = 13
```

Comment:
?

Reply:
Currently in master repository of Kubernetes the Vietnamese and Russian language have the same weight 11!!!

Reply:
Sorry for the confusion. I do see that there is duplication and did not realize the ordering of the
weights.